cmd/pillard/main.go

```diff
@@ -0,0 +1,40 @@
+// Pillar is a web service for delivering community experiences
+package main
+
+import (
+	"runtime"
+
+	"github.com/ardanlabs/kit/log"
+	"github.com/ardanlabs/kit/web/app"
+
+	"github.com/coralproject/xenia/cmd/pillard/handlers"
+	"github.com/coralproject/xenia/cmd/pillard/routes"
+)
+
+// These are set by the makefile with:
+// go build -ldflags "-X main.GitVersion=8e830ff -X main.GitRevision=123123123123 -X main.BuildDate=2016-01-25"
```

Comment:
@edefikh Do we have this git revision info available at the time to building?  These flags are carried through to an api versions endpoint, which would be very helpful for troubleshooting deployments.